runtime/vm/jvminit.c

```diff
@@ -6360,6 +6360,13 @@ registerOSHandler(J9JavaVM *vm, U_32 signal, void *newOSHandler, void **oldOSHan
 	return rc;
 }
 
+BOOLEAN
+isMasterSignalHandler(J9JavaVM *vm, void *osHandler)
+{
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	return j9sig_is_master_signal_handler(osHandler);
```

Comment:
This is just a port library call, does it really need to be an internal function?

Reply:
Guess the same question applies to registerPredefinedHandler() and registerOSHandler().

Reply:
just verified that i can directly call j9sig_* functions from jvm.c. will either remove the wrappers or move the wrappers in jvm.c.

Reply:
I removed isMasterSignalHandler.

I can't move registerPredefinedHandler since predefinedHandlerWrapper and signalDispatch can't be moved to jvm.c.

we should keep registerOSHandler since it will help in uniformly registering signal handlers within the JVM. it may help in resolving [Unclear use of OMRSIG_SIGNAL / OMRSIG_SIGACTION)](https://github.com/eclipse/openj9/issues/1889) (OpenJ9 issue).